Library/Aliases/bladerf

```diff
@@ -0,0 +1 @@
+../Formula/libbladerf.rb
```

Comment:
Is this necessary? `brew search bladerf` would give you `libbladerf` since it’s included in the name.